src/Microsoft.Kusto.ServiceLayer/ObjectExplorer/DataSourceModel/NodePathGenerator.cs

```diff
@@ -24,7 +24,7 @@ internal static void Initialize()
             }
 
             var assembly = typeof(ObjectExplorerService).Assembly;
-            var resource = assembly.GetManifestResourceStream("Microsoft.Kusto.ServiceLayer.ObjectExplorer.SmoModel.TreeNodeDefinition.xml");
```

Comment:
This fixes a run time error caused by the folder being renamed from SmoModel to DataSourceModel